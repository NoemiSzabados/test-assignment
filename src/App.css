.App {
  text-align: center;
}

.container {
  margin: auto;
  display: flex;
}

@media only screen and (min-width: 1000px) {
  .container {
    max-width: 90%;
  }
}