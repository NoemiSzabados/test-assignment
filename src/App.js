import './App.css';

function App() {
  return (
    <div className="App">
      <h1>Word Cloud</h1>
    </div>
  );
}

export default App;
