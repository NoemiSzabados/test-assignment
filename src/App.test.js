import App from './App';
import ReactDOM from "react-dom"
import renderer from 'react-test-renderer';
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import WordCloudContainer from "./views/wordCloud";
import TopicInformation from "./views/topicInformation";

Enzyme.configure({ adapter: new Adapter() });

describe('App', () => {
    it("renders without crashing", () => {
        const div = document.createElement("div");
        ReactDOM.render(<App />, div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('snapshot test', () => {
        const component = renderer.create(
            <App />,
        );
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

    it("title and components displayed", () => {
        const wrapper = shallow(<App />);
        const text = wrapper.find("h1").text();

        expect(text).toEqual("Word Cloud");
        expect(wrapper.contains(<WordCloudContainer />)).toBe(true);
        expect(wrapper.contains(<TopicInformation />)).toBe(true);
    });
})
