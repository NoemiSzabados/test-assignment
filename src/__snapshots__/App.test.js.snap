// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot test 1`] = `
<div
  className="App"
>
  <h1>
    Word Cloud
  </h1>
  <div
    className="container"
  >
    <div
      className="wordCloudWrapper"
    >
      <div
        className="wordCloudContainer"
      />
    </div>
    <div
      className="infoWrapper hidden"
    />
  </div>
</div>
`;
